docs/chat2/DATABASE_ERD.md

```
@@ -0,0 +1,804 @@
+        varchar profile_image_url
+        varchar travel_style
+        varchar budget_level
+        varchar status
+        timestamp created_at
+        timestamp updated_at
+        timestamp deleted_at
+    }
+    
+    user_preferences {
+        bigint id PK
+        bigint user_id FK
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)